### DISCUSSION TITLE
Kwokctl failed to create cluster by default image when runtime is kind

### DISCUSSION THREAD

**Discussion by Fish-pro** [kubernetes-sigs/kwok#123] [Q&A] [answered]
**Please provide an in-depth description of the question you have**:

When you run the following command to install the cluster, the installation fails
```sh
➜  kwok kwokctl create cluster --name kwok1 --runtime kind
Creating cluster                                                   cluster=kwok1
Pull image                    cluster=kwok1 image=docker.io/kindest/node:unknown
Error response from daemon: manifest for kindest/node:unknown not found: manifest unknown: manifest unknown
ERROR Failed to setup configcluster=kwok1 err="docker pull docker.io/kindest/node:unknown: exit status 1"
Cluster is cleaned up                                              cluster=kwok1
ERROR Execute exit err="docker pull docker.io/kindest/node:unknown: exit status 1"
```

**What do you think about this question?**:

The cause is that the values of environment variables `KWOK_KUBE_VERSION` and `KWOK_VERSION` are read as image tags by default. If the environment variables do not exist, the following reference is used，when pull an image, the image does not exist
```sh
var (
	Version      = "unknown"
	KubeVersion  = "unknown"
	ImagePrefix  = "registry.k8s.io/kwok"
	BinaryPrefix = "https://github.com/kubernetes-sigs/kwok/releases/download"
	BinaryName   = "kwok-" + runtime.GOOS + "-" + runtime.GOARCH
)
```

So I understand is it possible to provide a default installation version when the user does not specify environment variables or parameters?

**Environment**:
kwokctl version: latest

**Comment by wzshiming** [comment, marked as answer]
refer to https://github.com/kubernetes-sigs/kwok/pull/94/files

We didn't actually release the latest tag, `docker.io/kindest/node` is the same as us.

```
$ skopeo list-tags docker://registry.k8s.io/kwok/kwok
{
    "Repository": "registry.k8s.io/kwok/kwok",
    "Tags": [
        "sha256-8f03c71a0c4b20c7b2ba00a5864efb85dc8cec948d880b3f6cf0d653d037e383.sig",
        "v0.0.1"
    ]
}
```

**Comment by Fish-pro** [comment]
> refer to https://github.com/kubernetes-sigs/kwok/pull/94/files
> 
> We didn't actually release the latest tag, `docker.io/kindest/node` is the same as us.
> 
> ```
> $ skopeo list-tags docker://registry.k8s.io/kwok/kwok
> {
>     "Repository": "registry.k8s.io/kwok/kwok",
>     "Tags": [
>         "sha256-8f03c71a0c4b20c7b2ba00a5864efb85dc8cec948d880b3f6cf0d653d037e383.sig",
>         "v0.0.1"
>     ]
> }
> ```

@wzshiming 
Thanks for your reply. I understand that my build method is wrong. I noticed that `KUBE_VERSION` and `VERSION` values are set in release.sh, so there is no problem
`hack/releases.sh`
```sh
  if [[ "${VERSION}" != "" ]]; then
    LDFLAGS+=("-X sigs.k8s.io/kwok/pkg/consts.Version=${VERSION}")
  fi
  if [[ "${KUBE_VERSION}" != "" ]]; then
    LDFLAGS+=("-X sigs.k8s.io/kwok/pkg/consts.KubeVersion=${KUBE_VERSION}")
  fi
```
